src/main/java/org/jabref/model/texparser/Citation.java

```diff
@@ -66,9 +66,9 @@ public String getContext() {
 
         // Add three dots when the string does not contain all the line.
         return String.format("%s%s%s",
-                (start > 0) ? "... " : "",
+                (start > 0) ? "..." : "",
```

Comment:
There is a Text overrun style for javafx 
Not sure if this helps or not 

https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/Tooltip.html#setTextOverrun-javafx.scene.control.OverrunStyle-

Reply:
Not here, because the `getContext()` method extracts a substring that always has the `\cite` command in the middle. That is what is shown into the dashed box.

But is could be useful in the near future. Thanks!